go.mod

```diff
@@ -20,7 +20,7 @@ require (
 	github.com/stretchr/testify v1.7.0
 	github.com/urfave/cli/v2 v2.2.0
 	github.com/vmware/govmomi v0.22.2-0.20200329013745-f2eef8fc745f
-	golang.org/x/net v0.0.0-20211015210444-4f30a5c0130f
+	golang.org/x/net v0.7.0
```

Comment:
Can you also change go version to 1.18 to be consistent with the plugin?

Reply:
Done.